components/Choose/index.tsx

```diff
@@ -55,8 +68,7 @@ export const Choose = () => {
       <form
         onSubmit={(e) => {
           e.preventDefault()
-          setSubmittedForm(true)
-          setMsigActor(safeID)
+          submitForm()
```

Comment:
In regards to https://github.com/glifio/safe/pull/132/files#r907683043 - could probably just use the `submittedForm` state var to conditionally render a load?

Reply:
The problem with that is that it doesn't reset after the load fails. Checking `errorMessage` for this might work once, but it's not very clean. Ideally, the `loading` property from `useMsig` should actually return whether it is currently loading an actor code or not.